symphony/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -0,0 +1,171 @@
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import type {lte_gateway} from '@fbcnms/magma-api';
+
+import Divider from '@material-ui/core/Divider';
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Paper from '@material-ui/core/Paper';
+import React from 'react';
+import SettingsIcon from '@material-ui/icons/Settings';
+import Text from '@fbcnms/ui/components/design-system/Text';
+import TextField from '@material-ui/core/TextField';
+
+import {makeStyles} from '@material-ui/styles';
+import {useState} from 'react';
+
+const useStyles = makeStyles(theme => ({
+  dashboardRoot: {
+    margin: theme.spacing(3),
+    flexGrow: 1,
+  },
+  paper: {
+    height: 100,
+    padding: theme.spacing(10),
+    textAlign: 'center',
+  },
+}));
+
+export default function GatewayConfig({gwInfo}: {gwInfo: lte_gateway}) {
+  const classes = useStyles();
+  return (
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={3}>
+        <Grid container spacing={3} item xs={12}>
+          <Text>
+            <SettingsIcon /> Config
+          </Text>
+        </Grid>
+        <Grid container spacing={3} item xs={6}>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item xs={6}>
+                <Text>Gateway</Text>
+              </Grid>
+              <Grid container item xs={6} justify="flex-end">
+                <Text>Edit</Text>
+              </Grid>
+            </Grid>
+            <GatewayInfoConfig readOnly={true} gwInfo={gwInfo} />
+          </Grid>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item xs={6}>
+                <Text>Aggregations</Text>
+              </Grid>
+              <Grid container item xs={6} justify="flex-end">
+                <Text>Edit</Text>
+              </Grid>
+            </Grid>
+            <Paper className={classes.paper} />
+          </Grid>
+        </Grid>
+        <Grid container spacing={3} item xs={6}>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item xs={6}>
+                <Text>EPC</Text>
+              </Grid>
+              <Grid container item xs={6} justify="flex-end">
+                <Text>Edit</Text>
+              </Grid>
+            </Grid>
+            <Paper className={classes.paper} />
+          </Grid>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item xs={6}>
+                <Text>Ran</Text>
+              </Grid>
+              <Grid container item xs={6} justify="flex-end">
+                <Text>Edit</Text>
+              </Grid>
+            </Grid>
+            <Paper className={classes.paper} />
+          </Grid>
+        </Grid>
+      </Grid>
+    </div>
+  );
+}
+
+function GatewayInfoConfig({
```

Comment:
minor nit: As i'm going through couple of other configuration components, i'm realizing that reusing the same component for edit and show is little hard since the views are different. For e.g. https://github.com/facebookincubator/magma/blob/21ed94d8dec91d619fa5a160912d8cf648794072/symphony/app/fbcnms-projects/magmalte/app/views/network/NetworkInfo.js Might be easier to simplify this and get rid of readOnly prop and use ListItemText everywhere.